main/webapp/modules/core/scripts/reconciliation/standard-service-panel.js

```diff
@@ -98,7 +99,10 @@ ReconStandardServicePanel.prototype._constructUI = function() {
   this._elmts.typeInput.attr('aria-label',$.i18n('core-recon/type'))
 
   this._elmts.rawServiceLink.attr("href", this._service.url);
-
+  fetch(this._service.url)
```

Comment:
Rather than fetching the manifest a second time we should pick up the link from the existing data, [looking at the registration I guess we already have it in there](https://github.com/OpenRefine/OpenRefine/blob/0ade8a310716d3115eef5ae711170e1d02358350/main/webapp/modules/core/scripts/reconciliation/recon-manager.js#L81).

Reply:
Hey, unable to get this ,isn't the link you are pointing to pushes the reconciliation services into the stack?

Reply:
Yes, but it should push as I read it the entire manifest object, including the documentation property.

Reply:
what do you think about this?
![Screen Shot 2023-04-19 at 12 53 39 AM](https://user-images.githubusercontent.com/72227873/232884192-b56a13e5-9578-4016-8ac8-e193308c29fe.png)

Reply:
Hi @ayushrai206, just a general comment about communication: please do not take screenshots of code, but rather submit that code in the pull request by committing and pushing it. Alternatively, you can use the "suggest changes" function to embed small code snippets in your PR comments. Or simply a code block in a comment. If none of those options work for you, it would still be better to share your code in a textual format (for instance by uploading it to another platform), shared in any other way.

We cannot copy a screenshot and put it in our local copy of the code to try it out. We cannot copy parts of it to quote them in our reply to you. If people use screen readers, they will also have a hard time. So please: no screenshots of textual content here.

Reply:
@ayushrai206 you are looking for the CSS "display" property rather than "visibility". You can also use jQuery's "hide()" method.

Reply:
@Abbe98 done!